frontend/src/app/Settings/Users/InProgressModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Modal from "react-modal";
+import Button from "../../commonComponents/Button";
+
+interface Props {
+  onClose: () => void;
+  onContinue: () => void;
+}
+
+const InProgressModal: React.FC<Props> = ({ onClose, onContinue }) => {
+  return (
+    <Modal
+      isOpen={true}
+      style={{
+        content: {
+          inset: "3em auto auto auto",
```

Comment:
Your call on this, but I've started to add a `className` to the `Modal` and then define these CSS properties in the corresponding `.scss` file. I think there may be some hoisting we can do for things common to all the modals such as the `-overlay` but the dialog itself always seems to need unique CSS. For an example see `TestResultCorrectionModal`.

Reply:
Yeah, reusable classnames are def the way to go.

My css-fu is failing me on some of these modals, esp to get them vertically/horizontally centered and responsive in both firefox AND chome. I'm mostly just doing copy-pasta of the AoE modal and making slight tweaks to the css just to move forward.